assets/data-port/import.js

```diff
@@ -28,6 +28,11 @@ const SenseiImportPage = () => {
 		fetchImporterData();
 	}, [ fetchImporterData ] );
 
+	// Add `sensei-color` to body tag.
+	useLayoutEffect( () => {
+		document.body.classList.add( [ 'sensei-color' ] );
```

Comment:
Not critical right now as navigating away will reload the page, but to be safe, a cleanup function should be returned that removes this, like:
`return () => document.body.classList.remove( [ 'sensei-color' ] );`

Reply:
TIL! Fixed it in rebase.